constants.py

```diff
@@ -24,6 +24,9 @@ class TraceData:
     VARNAME = "VarName"
     VARTYPE_MODULE = "TypeModule"
     VARTYPE = "Type"
+    VARTYPE2 = "Type 2"
```

Comment:
Perhaps a better name? First one is "original", second is "generated"?

Reply:
I can't change the name of the first VARTYPE as it is also used in Trace Data & Type Hint Data. I also can't change the name of VARTYPE2 to Generated Type as the  VARTYPE as that is used in Trace Data & Type Hint Data corresponds to the generated type (just not for the metrics data). 

However, if you find more fitting names, I will consider changing the names.

Reply:
Then create two new variables, so that you can name them freely without being dependent on the names of the DataFrames for tracing

Reply:
Done